core/src/components/datetime/datetime.tsx

```diff
@@ -1213,12 +1243,13 @@ export class Datetime implements ComponentInterface {
     ampmItems: PickerColumnItem[],
     use24Hour: boolean
    ) {
-    const { color, workingParts } = this;
+    const { color, activePartsClone: timeParts } = this;
+
     return (
       <ion-picker-internal>
         <ion-picker-column-internal
           color={color}
-          value={workingParts.hour}
+          value={timeParts.hour}
```

Comment:
Small nitpick, but this object isn't just time parts anymore, so maybe we should call it `activePartsClone`?

Reply:
This should be good now 👍  I just went ahead and removed the rename and used destructing assignment for all variables in the render function.